commands/user

```diff
@@ -257,3 +257,80 @@ function cmd_set_role {
   log_audit "Set role ${role} for user ${name}"
   echo "User ${name} now has ${role} privileges."
 }
+
+# helper function for read and check user names
+function get_names {
+  local names=()
```

Comment:
It's better to not combine `local` with initialization. In this particular case it doesn't make any difference, but if you cared for the completion code of the initialization, then `local` on the same line would clobber it.

Also, if you're to define an array, please use `local -a` just like you would do with `declare -a`.